effekt/jvm/src/main/scala/effekt/Backend.scala

```diff
@@ -27,7 +27,7 @@ case class Backend[E](name: String, compiler: Compiler[E], runner: Runner[E])
 object Backend {
 
   def backend(name: String): Backend[_] = name match {
-    case "js"           => Backend("js", js.JavaScript(), JSRunner)
+    case "js"           => Backend("js", js.JavaScript(List("jsNode")), JSRunner)
```

Comment:
What if I want to generate code which is designed to run in the browser (like my presentation tool), wouldn't I need `jsWeb` here too? 🤔

Reply:
Hm, currently AFAICT there is no web/node distinction when generating code (except that IIUC the website uses the scala.js-compiled Compiler? Or doesn't it?)

One way to solve this would be to add a separate `js-web` backend. Alternatively, we could drop this PR and instead do a runtime check for node/browser.